templates/plugin/python/module.conf.erb

```diff
@@ -11,7 +11,11 @@
     <%= key %> <%= k %> <% if !!v == v %><%= v %><% else %>"<%= Shellwords.split(v).join('" "') %>"<% end %>
   <%- end -%>
   <%- else -%>
-    <%= key %> <% if !!value == value %><%= value %><% else %>"<%= Shellwords.split(value).join('" "') %>"<% end %>
+  <%- if not !!value == value and value.include?('"') -%>
```

Comment:
tests passed. Should I revert the modules.conf.erb back to current master and push to trigger the builds again? Curious if the jruby buld slaves catch the failures ( would leave my tests in place )